.gitignore

```diff
@@ -3,3 +3,5 @@
 /doc
 /tmp
 erl_crash.dump
+
+.elixir_ls/
```

Comment:
this seems like it is going to end up as a pretty commonly gitignored folder, as elixir_ls gets more popular.

Reply:
Anything that is specific to an editor should be in your global gitignore.

Reply:
I've been doing this for way too long not to have known about the global gitignore :D 🤦‍♂️